crypto/evp_extra/print.c

```diff
@@ -306,6 +308,48 @@ static int eckey_priv_print(BIO *bp, const EVP_PKEY *pkey, int indent) {
   return do_EC_KEY_print(bp, pkey->pkey.ec, indent, 2);
 }
 
+// Dilithium keys.
+
+static int do_dilithium_print(BIO *bp, const EVP_PKEY *x, int off, int ptype) {
+  if (x == NULL) {
+    OPENSSL_PUT_ERROR(EVP, ERR_R_PASSED_NULL_PARAMETER);
+    return 0;
+  }
+
+  if (!BIO_indent(bp, off, 128)) {
+    return 0;
+  }
+
+  DILITHIUM3_KEY *key = x->pkey.ptr;
+  int bit_len = 0;
+
+  if (ptype == 2) {
```

Comment:
Can this use a shared constant here and in pub/priv_print?

Reply:
Following syntax of the rest of the printing procedures for constancy and readability.